packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -552,13 +564,11 @@ class InboundReferencesWidget extends StatelessWidget {
   String _inboundRefDescription(InboundReference inboundRef, int? offset) {
     if (inboundRef.parentListIndex != null) {
       final ref = inboundRef.source;
-      if (ref is InstanceRef) {
-        return 'Referenced by element [${inboundRef.parentListIndex}] of '
-            '${ref.classRef?.name ?? '<parentListName>'}';
-      } else {
-        return 'Referenced by element [${inboundRef.parentListIndex}] of '
-            '${_objectName(ref) ?? '<unknown>'}';
-      }
+      final parentListIndex = inboundRef.parentListIndex;
+
+      final parentListName = _containerClass(ref) ?? '<parentListName>';
+
+      return 'Referenced by element [$parentListIndex] of $parentListName';
```

Comment:
this code is still duplicated from above. Put into a helper,

`_parentListIndexDescription(int parentListIndex)` or something like that.

```dart
String _parentListIndexDescription(int parentListIndex) {
   final parentListName = _containerClass(ref) ?? '<parentListName>';      
   return 'Referenced by element [$parentListIndex] of $parentListName';
}
```

Reply:
This one says referenced instead of retained.
Would this be ok(?) :
```
String _parentListIndexDescription(int parentListIndex, ObjRef ref) {
   final parentListName = _containerClass(ref) ?? '<parentListName>';      
   return ' by element [$parentListIndex] of $parentListName';
}
```

Reply:
yep looks good